Scripts/Input/OVRTouchInputToEvents.cs

```diff
@@ -1,164 +1,167 @@
-﻿using System;
-using UnityEngine;
-using UnityEngine.InputNew;
-using UnityEngine.Experimental.EditorVR;
+﻿#if UNITY_EDITOR
+using System;
+using UnityEditor.Experimental.EditorVR;
 
 [assembly: OptionalDependency("OVRInput", "ENABLE_OVR_INPUT")]
 
-/// <summary>
-/// Sends events to the input system based on native Oculus SDK calls
-/// </summary>
-public class OVRTouchInputToEvents : MonoBehaviour
+namespace UnityEngine.InputNew
```

Comment:
s this namespace correct? Shouldn't it be _UnityEngine.Experimental._**something**_.Input_ ?